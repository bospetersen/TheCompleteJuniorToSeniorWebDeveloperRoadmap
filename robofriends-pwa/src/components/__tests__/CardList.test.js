import Enzyme, { shallow, mount, render } from 'enzyme';
import React from 'react';
import CardList from '../CardList';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';

Enzyme.configure({ adapter: new Adapter() });

// it('Expect to render CardList component 1 time', () => {
//   expect(shallow(<CardList />).length).toEqual(1)
// });

const mockRobots = [{
  id: 1,
  name: 'John Snow',
  username: 'JohnSnow',
  email: 'johndoe@gmail.com'
}]

it('Takes snapshot of CardList component', () => {
  expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
})