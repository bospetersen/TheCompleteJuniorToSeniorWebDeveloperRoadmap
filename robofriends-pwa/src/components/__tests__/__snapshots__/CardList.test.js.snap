// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Takes snapshot of CardList component 1`] = `
<div>
  <Card
    email="johndoe@gmail.com"
    id={1}
    key="0"
    name="John Snow"
  />
</div>
`;
